easybuild/easyconfigs/h/HDF5/HDF5-1.10.1-intel-2018b.eb

```diff
@@ -0,0 +1,25 @@
+name = 'HDF5'
+version = '1.10.1'
+
+homepage = 'https://support.hdfgroup.org/HDF5/'
+description = """HDF5 is a data model, library, and file format for storing and managing data.
+ It supports an unlimited variety of datatypes, and is designed for flexible
+ and efficient I/O and for high volume and complex data."""
+
+toolchain = {'name': 'intel', 'version': '2018b'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+source_urls = ['https://support.hdfgroup.org/ftp/HDF5/releases/hdf5-%(version_major_minor)s/hdf5-%(version)s/src']
+sources = [SOURCELOWER_TAR_GZ]
+patches = ['HDF5-%(version)s-mpiifort-2018.0.128-fix.patch']
```

Comment:
without the patch, 2018.3.222 doesn't seem to throw the error anymore, but since it says the commented out lines are unnecessary and violate the standard, I suppose the patch should be kept anyway (?)

Reply:
The patch no longer applies for HDF5 1.10.2, so I removed it...